### PR TITLE
Updates linter file

### PR DESCRIPTION
**Pull request by kathyoleary**
📗 Ready to Go,  Minor Linter updates!  Thanks

### CODE REVIEW

**Comment by rylnd** [comment]
@kathyoleary can you please update the existing linter rules from severity `2` to severity `"error"` ?

**Comment by rylnd** [comment]
@kathyoleary it looks like the current build errors are due to a syntax error in the eslintrc file.

File: .eslintrc.js [hunk 1]
```diff
@@ -9,21 +9,23 @@ module.exports = {
...
     "quotes": [
-      2,
+      "error",
       "single"
     ],
     "semi": [
-      2,
+      "error",
       "always"
     ]
```
Comment (camwiegert) [inline .eslintrc.js]: Missing a comma here.

**Comment by rylnd** [comment]
I guess I was wrong: you can't interchange `2` with `"error"` in the rule severity.

**Comment by kathyoleary** [comment]
Actually, that one is in there EXACTLY how they show it on the Eslint page.

**Comment by kathyoleary** [comment]
FYI, cannot have "error" on anyone of them, has to be 2

**Comment by rylnd** [comment]
LGTM 👍

**Comment by kathyoleary** [comment]
In updating the linter file and fixing it, I also added the recommended from:  https://github.com/LMHT/LMHT.github.io/issues/180
And I went through and added no multiple empty lines, and no unnecessary nested blocks.  The spaces in the parentheses is already covered because the linter has the recommended rules already loaded. 

So if Erica agrees, I'd like to close https://github.com/LMHT/LMHT.github.io/issues/180 when we close this one.

**Comment by ericanaughton** [comment]
👍

**Pull request merged**